backend/src/test/java/com/cruru/question/controller/QuestionControllerTest.java

```diff
@@ -1,48 +1,81 @@
 package com.cruru.question.controller;
 
+import static org.springframework.restdocs.cookies.CookieDocumentation.cookieWithName;
+import static org.springframework.restdocs.cookies.CookieDocumentation.requestCookies;
+import static org.springframework.restdocs.payload.PayloadDocumentation.fieldWithPath;
+import static org.springframework.restdocs.payload.PayloadDocumentation.requestFields;
+import static org.springframework.restdocs.request.RequestDocumentation.parameterWithName;
+import static org.springframework.restdocs.request.RequestDocumentation.queryParameters;
+import static org.springframework.restdocs.restassured.RestAssuredRestDocumentation.document;
+
 import com.cruru.applyform.domain.ApplyForm;
 import com.cruru.applyform.domain.repository.ApplyFormRepository;
-import com.cruru.dashboard.domain.Dashboard;
-import com.cruru.dashboard.domain.repository.DashboardRepository;
+import com.cruru.choice.controller.dto.ChoiceCreateRequest;
+import com.cruru.choice.domain.repository.ChoiceRepository;
 import com.cruru.question.controller.dto.QuestionCreateRequest;
 import com.cruru.question.controller.dto.QuestionUpdateRequests;
 import com.cruru.question.domain.QuestionType;
 import com.cruru.question.domain.repository.QuestionRepository;
 import com.cruru.util.ControllerTest;
 import com.cruru.util.fixture.ApplyFormFixture;
-import com.cruru.util.fixture.DashboardFixture;
 import com.cruru.util.fixture.QuestionFixture;
 import io.restassured.RestAssured;
 import io.restassured.http.ContentType;
 import java.util.List;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.restdocs.payload.FieldDescriptor;
 
 @DisplayName("질문 컨트롤러 테스트")
 class QuestionControllerTest extends ControllerTest {
 
+    private static final FieldDescriptor[] QUESTION_FIELD_DESCRIPTORS = {
+            fieldWithPath("type").description("질문의 유형"),
+            fieldWithPath("question").description("질문 내용"),
+            fieldWithPath("choices").description("질문의 선택지들"),
+            fieldWithPath("orderIndex").description("질문의 순서"),
+            fieldWithPath("required").description("질문의 필수 여부")
+    };
+
+    private static final FieldDescriptor[] CHOICE_FIELD_DESCRIPTORS = {
+            fieldWithPath("choice").description("선택지의 내용"),
+            fieldWithPath("orderIndex").description("선택지의 필수 여부")
```

Comment:
```suggestion
            fieldWithPath("orderIndex").description("선택지의 순서")
```
field 명과 설명이 불일치 합니다.